tools/clang/test/CodeGenSPIRV/fn.ctbuffer.hlsl

```diff
@@ -0,0 +1,23 @@
+// Run: %dxc -T ps_6_0 -E main
+
+// CHECK: %type_MyCBuffer = OpTypeStruct %v4float
+// CHECK: %type_MyTBuffer = OpTypeStruct %float
+cbuffer MyCBuffer {
+    float4 cb_val;
+
+// CHECK:      %get_cb_val = OpFunction %v4float None %17
```

Comment:
oops. %17

Reply:
Ha, I clearly remembered I've fixed this... Seems bad rebasing or something.

Will fix. Thanks for pointing out!

Reply:
Done